src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -144,42 +148,36 @@ enum SideMenuButtonType {
         }
 
         let backLabel = setupLabel(title: kLocalizedBack, selector: #selector(delegate?.stopAction), type: .landscapeFirstColumn)
-        backLabel.topAnchor.constraint(equalTo: backButton.bottomAnchor, constant: marginLabel).isActive = true
+        backLabel.topAnchor.constraint(equalTo: backButton.bottomAnchor).isActive = true
 
-        let continueButton = setupButton(imageName: "stage_dialog_button_continue", selector: #selector(delegate?.continueAction), type: .landscapeSecondColumn)
-        if #available(iOS 11.0, *) {
-            continueButton.topAnchor.constraint(equalTo: self.safeAreaLayoutGuide.topAnchor, constant: marginTopBottom).isActive = true
-        } else {
-            continueButton.topAnchor.constraint(equalTo: self.topAnchor, constant: marginTopBottom).isActive = true
-        }
+        let continueButton = setupButton(imageName: "stage_dialog_button_continue", selector: #selector(delegate?.continueAction), type: .landscapeFirstColumn)
```

Comment:
Please change the enum values to more meaningful names (e.g., `landscapeLarge`, `landscapeSmall` or `landscapeTopRow`, `landscapeBottomRow`).